salt/modules/pillar.py

```diff
@@ -189,7 +189,7 @@ def get(
     return ret
 
 
-def items(*args, **kwargs):
+def items(*args, pillar=None, pillar_enc=None, pillarenv=None, saltenv=None):
```

Comment:
Did not want for you to update this one yourself(since it's not scoped to your fix) but I'm quite happy that you did. Thank You!